### DISCUSSION TITLE
After installing and successfully using localhost:3000 for the training it no longer functions (spins) ideas?

### DISCUSSION THREAD

**Discussion by rkobetich** [cypress-io/cypress#30680] [Questions and Help]
I have successfully installed,  launched the server via npm run dev
and then launching cypress via npx cypress open
it works in cypress but when I closed the cypress sessions,  closed the terminals,  then did the same to restart the server no longer functions.  How do I resolve?  I've rebooted, restarted everything to no avail.  This makes it impossible to continue the cypress training.  I'm trying to evaluate for my company and it's getting hard to use.  Thanks for ANY help :-)  I also did npm restart dev

**Comment by matheusvianaultralims** [comment]
Can you send us the error? And when you restart your PC it still doesn't work?
